resources/user-agents/apps/facebook/facebook-app.json

```diff
@@ -195,22 +195,21 @@
             "N9519": "ZTE N9519",
             "Nexus 5": "LG Nexus 5",
             "R1 HD": "BLU R1 HD",
-            "SAMSUNG-SM-G891A": "Samsung SM-G891A",
```

Comment:
Interesting, is this how we should deal with Samsung devices that prefix with `SAMSUNG-` in the future?

Any concern that the `#PLATFORM#` replacement won't end with an asterisk, or is it safe to assume that it always will?